addons/website_slides/views/website_slides_templates_utils.xml

```diff
@@ -152,7 +152,7 @@
             <i t-if="slide_completed" class="o_wslides_slide_completed fa fa-check-circle fa-fw text-success fa-lg" t-att-data-slide-id="slide.id" title="Mark as not done"/>
             <i t-else="" t-attf-class="fa #{uncompleted_icon} fa-fw fa-lg" t-att-data-slide-id="slide.id" title="Mark as done"/>
         </button>
-        <button class="o_wslides_button_complete o_wslides_button_uncompleted btn btn-sm" t-else="" disabled="1">
+        <button class="o_wslides_button_complete o_wslides_button_uncompleted btn btn-sm border-0" t-else="" disabled="1">
```

Comment:
Should we do the same change here then?
https://github.com/odoo/odoo/blob/saas-17.3/addons/website_slides/static/src/xml/website_slides_sidebar.xml#L16